src/libraries/icubmod/canBusSkin/canBusSkin.ini

```diff
@@ -1,5 +0,0 @@
-[plugin canBusSkin]
-type device
-name canBusSkin
-library canBusSkin
-wrapper skinwrapper
```

Comment:
Warning: the `yarp_prepare_plugin` method uses `grabber`

Reply:
fixed to skinwrapper